### ISSUE TITLE
Entities Mapping

### ISSUE THREAD

**Issue by jaroslavkozak** [wortell/AZSentinel#182]
Will it be possible to assign entities using a script and/or function?

Will it be possible to export the settings for future replication?

**Comment by gabrielnecula** [comment]
I'm also in a big need of this. Mandatory for using this in CICD.

**Comment by alvaro354** [comment]
Yes it could be good if that feature is added in the next update. Entity Mapping and the new Alert details enrichment

**Comment by sandeep5234** [comment]
Any update on this please? As part of our threat hunting we need Entities mapped to alerts. At the moment azsentinel module doesn't import entities.

**Comment by sandeep5234** [comment]
@pkhabazi  I am thinking to write my own PowerShell function that will use Azure sentinel Analytical rules creation API.  Could you please let me know if entities deployment feature will be added soon? If yes then I won't spend time writing my own code. 
Thank you.

**Comment by pavelmirlenko** [comment]
Any update? Entity mappings are important part of Analytics rule. Thank you for your efforts!
